tagstudio/src/core/ts_core.py

```diff
@@ -7,51 +7,9 @@
 import json
 import os
 
+from src.core.constants import TEXT_FIELDS
```

Comment:
Also needs `TS_FOLDER_NAME`

```suggestion
from src.core.constants import TEXT_FIELDS, TS_FOLDER_NAME
```